### PR TITLE
Translated up to (but not including) the credits. 

### PR DESCRIPTION
**Pull request by jkoelling**
Needs to be edited further to fit the available space and to match with the already translated parts.

You can just leave this open until I've proof read everything and fixed the layout issues. I just wanted to get the pull request out early so that the work I did is visible to you and anyone else that might want to chip in.

(I also have some fixes and suggestions for the already translated parts, but I'll create a different pull request for those once I'm done with this one.)

### CODE REVIEW

File: de.html [hunk 1]
```diff
@@ -1309,31 +1309,31 @@
 <panel w=500 h=350>
 <pic src="pics/end0.png" sx=0 sy=0></pic>
 <words x=10 y=10 w=450 h=60>
-	To help you get started with Spaced Repetition <i>today</i>,
-	you need to answer four questions:
+	Um dir zu helfen <i>heute</i> mit Spaced Repetition anzufangen,
```
Comment (jkoelling) [inline de.html]:
```suggestion
	Um dir zu helfen <i>heute</i> mit Intervall-Lernen anzufangen,
```
Der Begriff scheint halbwegs etabliert, macht Sinn und passt von der Länge her. Ich schlage vor bei der ersten Verwendung ne Fußnote auf das lange "Wiederholungen in regelmäßigen Abständen" einzufügen.

Comment (stellingsimon) [reply de.html]: "Intervall-Lernen" find ich super!

**Comment by stellingsimon** [comment]
Cool, thanks for helping out!

> You can just leave this open until I've proof read everything and fixed the layout issues. I just wanted to get the pull request out early so that the work I did is visible to you and anyone else that might want to chip in.

Ok, I'll do that. I don't expect to be able to do any active translating in the coming week, but I'll make sure to at least proof-read your contributions :-)

> (I also have some fixes and suggestions for the already translated parts, but I'll create a different pull request for those once I'm done with this one.)

Separate PR/included here is fine, whatever suits you :-) You may have noticed that I systematically omitted the "ß" -- that's because I'm Swiss and don't know where to put them. If you could fix those issues, that'd be great! Also, feel free to mess with anything I translated. This is the first time I'm translating a text and I'm sure there's lots of space for improvement. Also, I don't really care for having my name up there in the credits; so feel free to replace mine with yours.

File: de.html [hunk 2]
```diff
@@ -1347,27 +1347,28 @@
...
-	a new instrument
+	ein neues Instrument
 </words>
 <words x=115 y=195 w=100 style="text-align:left" no-bg class="comic_text">
-	computer programming
+	Programmierung
 </words>
 <words x=340 y=195 w=150 style="text-align:left" no-bg class="comic_text">
-	details of friends' lives
+	Details über die Leben deiner Freunde
```
Comment (stellingsimon) [inline de.html:1362]:
```suggestion
	Details über das Leben deiner Freunde
```

File: de.html [hunk 3]
```diff
@@ -1443,51 +1444,54 @@
...
+	(Du möchtest was anderes?
+	Hier ein paar andere Tools:	
 	<a target="_blank" href="https://www.supermemo.com/">SuperMemo</a>,
 	<a target="_blank" href="https://www.nimblenotes.com/">NimbleNotes</a>,
 	<a target="_blank" href="https://mnemosyne-proj.org/">Mnemosyne</a>)
 </words>
 <words x=110 y=620 w=360 h=40 no-bg fontsize=20>
-	* <i>anti</i>-disclaimer: i am <i>not</i> affiliated with any of these.
-	i just think they're cool &amp; helpful!
+	* <i>Anti</i>-Haftungsausschluss: Ich bin mit <i>keinem</i> davon geschäfftlich verbunden.
```
Comment (stellingsimon) [inline de.html:1487]: Typo in `geschäfftlich`: richtig wäre `geschäftlich`

File: de.html [hunk 4]
```diff
@@ -1700,64 +1707,63 @@
...
 
 <panel w=500 h=250>
 <pic src="pics/end1.png" sx=480 sy=500></pic>
 <words x=160 y=30 w=200 no-bg>
-	I'll miss the time we had together...
+	Ich werde unsere gemeinsame Zeit vermissen...
 </words>
 <words x=200 y=120 w=250 no-bg>
-	...but I hope we live on in each others' memories!
+	...aber ich hoffe, dass wir in unseren Erinnerungen fortleben!
```
Comment (stellingsimon) [inline de.html:1728]: `weiterleben` statt `fortleben`?

**Comment by jkoelling** [comment]
I'll try answer to your suggestions  and finish this sometime during the week, thanks for the feedback so far!

**Comment by stellingsimon** [comment]
@jkoelling I *might* be able to continue working on this tomorrow... Have you made any progress that is not yet pushed? Just asking so I don't cause conflict and duplicate work.

**Pull request merged**